src/types/value/slice.ts

```diff
@@ -1,7 +1,7 @@
 import type { AnyRegularField } from "./types";
 
 /**
- * Slice - Sections of your website
+ * A Slice - sections of your website.
```

Comment:
@angeloashmore Could we change this to "sections of your webpages"?

To me, a section of a website is a group of pages. A section of a webpage is a group of elements.

Reply:
Done!